### DISCUSSION TITLE
Spider-Man reading lists incongruencies

### DISCUSSION THREAD

**Discussion by blyat** [DieselTech/CBL-ReadingLists#8] [Reading Lists]
I've found 2 small timeline incongruencies in the following reading lists so far:

- Spider-Man 001 (Silver Age).cbl
- Spider-Man 002 (The 70s).cbl

For the first list, the issue **The X-Men #35** should go after **The Amazing Spider-Man # 52** instead of **The Amazing Spider-Man # 50**. This is because there are references to the Kingpin issues (**The Amazing Spider-Man # 51 and # 52**).

For the second list, the issue **Marvel Team-Up # 4** should go after **Marvel Team-Up # 3** instead of going after **The Amazing Spider-Man # 105**. This is because it is currently breaking the story flow that happens between **The Amazing Spider-Man # 104** and **# 105**, and also because it belongs with the small Morbius "arc" which is 5 issues including **The Amazing Spider-Man # 100 - # 102** and **Marvel Team-Up # 3** and **# 4**.

**Comment by DieselTech** [comment]
I just wanted to let you know I've acknowledged the issue but it might take a bit to get around to changing things. Hopefully not too long but we are still in the ramp up phase of things and figuring out the best way to go forward.

**Reply by jakem742** [reply]
Do we want to restrict the discussion area for now to not include discussions over issue order? Big picture discussion (ie. where events and whole series fit in) might make sense to resolve early, but shuffling individual issues around seems like something we do when we get to the details stage.

**Comment by flips22** [comment]
Thanks for the comment.  I just checked the new architecture that we are working on and both issues you brought up are fixed on there already.  As Diesel said it will be a bit before we get it out, but we are working on it.
